Dx diagram new api descriptions (#416)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md

---
---
##### shortDescription

> Raised after a custom command item has been clicked and allows you to perform custom actions.

Reviewer: ```suggestion
Raised after a custom command item was clicked and allows you to perform custom actions.
```

Reviewer: "to perform custom actions" this is a bit vague

Author: perhaps: "to implement the custom command's logic."

Reviewer: Ok
